src/lib/components/FormFeedback.js

```diff
@@ -94,17 +93,12 @@ class DisconnectedFormFeedback extends Component {
         <Grid container>
           <Grid.Column width={15} textAlign="left">
             <Message.Header>{message}</Message.Header>
-            <Message.List>{listErrors}</Message.List>
+            { Boolean(listErrors.length) && <Message.List>{listErrors}</Message.List> }
```

Comment:
`listErrors.length && ...` should be enough, no?

Reply:
I thought so too at first! But I learned that React only doesn't render `false` values. So in the case of `listErrors.length && ...`, React renders `0` if the list is empty. Therefore the need for the `Boolean`.

Reply:
I pushed `listErrors.length > 0` to be less weird :)

Reply:
Interesting didn't know that! Thanks for the explanation :)